### PR TITLE
Feat/donation buttons

### LINKED ISSUE #229: 🚀 Feature Request: Add donation buttons to the app
### Problem Overview 👁️‍🗨️ 

Users don't currently have a way to donate to Grey Software through the Toonin app

### What would you like? 🧰

Donation buttons for Paypal, Github, and OpenCollective on the header next to the dark mode and repo buttons. 

### What alternatives have you considered? 🔍

N/A

### Additional details ℹ️

OpenCollective link: https://opencollective.com/grey-software
Github Sponsors link: https://github.com/sponsors/grey-software
Paypal link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=VEAGAZP7DHJNE&source=url

### PR DESCRIPTION
**Pull request by patrickmonteiro**
Fixes #229 

So that the layout didn't get too polluted and full of buttons, I decided to make a new donation tab. Inside it containing the specific button for all types of donations.

Captura de Tela 2020-10-29 às 21 10 04

### CODE REVIEW

**Comment by diracs-delta** [comment]
looks amazing! :+1:

**Comment by ArsalaBangash** [comment]
@patrickmonteiro This looks excellent! I'd recommend changing the OpenCollective label to OpenCollective and adding the OpenCollective icon. 

You can see how icons are added in ``/packages/app/src/App.vue``

### OpenCollective Light

```
<svg width="32" height="32" viewBox="0 0 32 32" fill="none"
    xmlns="http://www.w3.org/2000/svg">
    <path fill-rule="evenodd" clip-rule="evenodd" d="M29.1531 6.8877C30.948 9.47379 31.9999 12.614 31.9999 16.0003C31.9999 19.3866 30.948 22.5271 29.1531 25.1129L25.0085 20.9684C25.8225 19.4957 26.2858 17.8019 26.2858 16.0003C26.2858 14.1987 25.8225 12.5052 25.0085 11.0325L29.1531 6.8877Z" fill="#8FC7FF"/>
    <path fill-rule="evenodd" clip-rule="evenodd" d="M25.1126 2.84685L20.9678 6.99138C19.4951 6.17745 17.8016 5.71417 16 5.71417C10.3194 5.71417 5.71418 10.3194 5.71418 16C5.71418 21.6806 10.3194 26.2858 16 26.2858C17.8016 26.2858 19.4951 25.8226 20.9678 25.0086L25.1126 29.1532C22.5265 30.948 19.3863 32 16 32C7.16352 32 0 24.8365 0 16C0 7.16351 7.16352 0 16 0C19.3863 0 22.5265 1.05197 25.1126 2.84685Z" fill="#297EFF"/>
</svg>
```

### OpenCollective Dark

```
<svg width="32" height="32" viewBox="0 0 32 32" fill="none"
    xmlns="http://www.w3.org/2000/svg">
    <path fill-rule="evenodd" clip-rule="evenodd" d="M29.1531 6.8877C30.948 9.47379 31.9999 12.614 31.9999 16.0003C31.9999 19.3866 30.948 22.5271 29.1531 25.1129L25.0085 20.9684C25.8225 19.4957 26.2858 17.8019 26.2858 16.0003C26.2858 14.1987 25.8225 12.5052 25.0085 11.0325L29.1531 6.8877Z" fill="#CCCCCC"/>
    <path fill-rule="evenodd" clip-rule="evenodd" d="M25.1126 2.84685L20.9678 6.99138C19.4951 6.17745 17.8016 5.71417 16 5.71417C10.3194 5.71417 5.71418 10.3194 5.71418 16C5.71418 21.6806 10.3194 26.2858 16 26.2858C17.8016 26.2858 19.4951 25.8226 20.9678 25.0086L25.1126 29.1532C22.5265 30.948 19.3863 32 16 32C7.16352 32 0 24.8365 0 16C0 7.16351 7.16352 0 16 0C19.3863 0 22.5265 1.05197 25.1126 2.84685Z" fill="#FFFFFF"/>
</svg>
```

File: packages/app/src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -61,6 +61,11 @@
         name="share"
         icon="mdi-access-point"
       />
+      <q-tab
+        label="Donate"
+        name="donate"
+        icon="mdi-currency-usd"
```
Comment (ArsalaBangash) [inline packages/app/src/layouts/MainLayout.vue:67]: Please use mdi-charity https://iconify.design/icon-sets/mdi/charity.html

File: packages/app/src/pages/DonatePage.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+<script>
+export default {
+  data: () => ({
+  }),
+}
+</script>
+
+<style scoped>
+.share-container {
```
Comment (ArsalaBangash) [inline packages/app/src/pages/DonatePage.vue:59]: donate-container

File: packages/app/src/pages/DonatePage.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  width: 599px;
+  height: auto;
+}
+
+.btn-share {
```
Comment (ArsalaBangash) [inline packages/app/src/pages/DonatePage.vue:64]: btn-donate

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Solid iteration! Please change the OpenCollective icons and address the review comments :)

File: packages/app/src/App.vue [hunk 4]
```diff
@@ -12,22 +12,26 @@
         style="width: 300px"
       >
         <q-card-section>
-          <div style="color:#343434;" class="text-h6">Are you using Chrome?</div>
+          <div
```
Comment (ArsalaBangash) [inline packages/app/src/App.vue:15]: I'd also recommend moving this dark mode fix to another PR. It'll help keep the PRs more single-featured and it'd help you out with Hacktoberfest :)

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
I suggest moving the dialog dark mode fix to a separate PR

File: packages/app/src/App.vue (hunk 4 above)
Comment (patrickmonteiro) [reply packages/app/src/App.vue:15]: its here: https://github.com/grey-software/toonin/pull/235

**Comment by patrickmonteiro** [comment]
all requests resolved. @ArsalaBangash

**Review by ArsalaBangash — APPROVED** [review]
Well done Patrick! :)
